.github/composite-actions/build-extensions/action.yml

```diff
@@ -32,10 +32,11 @@ runs:
         cd contrib/babelfishpg_money
         make -j 4 && make install
         cd ../babelfishpg_common
-        make -j 4 && make install
+        PG_CPPFLAGS='-DENABLE_SPATIAL_TYPES' make -j 4
+        PG_CPPFLAGS='-DENABLE_SPATIAL_TYPES' make install
         cd ../babelfishpg_tds
         make -j 4 && make install
         cd ../babelfishpg_tsql
-        PG_CPPFLAGS='-I/usr/include -DENABLE_TDS_LIB' SHLIB_LINK='-lsybdb -L/usr/lib64' make
-        PG_CPPFLAGS='-I/usr/include -DENABLE_TDS_LIB' SHLIB_LINK='-lsybdb -L/usr/lib64' make install
+        PG_CPPFLAGS='-I/usr/include -DENABLE_TDS_LIB -DENABLE_SPATIAL_TYPES' SHLIB_LINK='-lsybdb -L/usr/lib64' make
+        PG_CPPFLAGS='-I/usr/include -DENABLE_TDS_LIB -DENABLE_SPATIAL_TYPES' SHLIB_LINK='-lsybdb -L/usr/lib64' make install
```

Comment:
We should add another test to ensure the source compiles correctly without using TDS FDW and SPATIAL TYPE flags. Otherwise, we may break things unknowingly.

Reply:
We will followup this separately in future PR although we have tested it locally without using these flags, but we would test to ensure this in future PR